irods-vfs-impl/src/main/java/org/irods/nfsrods/vfs/IRODSVirtualFileSystem.java

```diff
@@ -1645,15 +1663,101 @@ private static Inode toFh(long _inodeNumber)
         return Inode.forFile(Longs.toByteArray(_inodeNumber));
     }
 
+    private String tryAndFindPathFromGenQuery(long _inodeNumber, String option) {
+        // if getPath() returns null, we'll use GenQuery to find the path
+        log_.debug("""
+                vfs::tryAndFindPathFromGenQuery
```

Comment:
The `vfs:` prefix was introduced to highlight methods that are part of the VFS interface.

This method should not use that prefix. Try looking at some of the other methods to see what I mean.

Reply:
Got it! Removed the `vfs::` prefix